src/Symfony/Component/Notifier/Bridge/Slack/SlackTransport.php

```diff
@@ -21,30 +21,31 @@
 
 /**
  * @author Fabien Potencier <fabien@symfony.com>
+ * @author Daniel Stancu <birkof@birkof.ro>
  *
  * @internal
  *
  * @experimental in 5.0
  */
 final class SlackTransport extends AbstractTransport
 {
-    protected const HOST = 'slack.com';
+    protected const HOST = 'hooks.slack.com';
 
-    private $accessToken;
-    private $chatChannel;
+    private $path;
 
-    public function __construct(string $accessToken, string $channel = null, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    protected $client;
+
+    public function __construct(string $path, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
     {
-        $this->accessToken = $accessToken;
-        $this->chatChannel = $channel;
+        $this->path = $path;
         $this->client = $client;
 
         parent::__construct($client, $dispatcher);
     }
 
     public function __toString(): string
     {
-        return sprintf('slack://%s?channel=%s', $this->getEndpoint(), $this->chatChannel);
+        return sprintf('%s://%s/%s', SlackTransportFactory::SCHEME, $this->getEndpoint(), $this->path);
```

Comment:
Is the `services/` part is always present, can we add it here instead of passing it in the path?